.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,58 @@
+name: Integration-Test
+on:
+  workflow_call:
+env:
+  GO_VERSION: "~1.18"
+  CERT_MANAGER_VERSION: "v1.10.0"
+  KUTTL_VERSION: "0.13.0"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  run-integration-test:
+    name: Run Integration Tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          cache: true
+          cache-dependency-path: '**/go.sum'
+
+      - name: Download manifests
+        uses: actions/download-artifact@v3
+        with:
+          path: ~/download/artifacts
+
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.3.0
+  
+      - name: Install cert-manager
+        run: |
+          kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/${{ env.CERT_MANAGER_VERSION }}/cert-manager.yaml
+          kubectl rollout status deployment cert-manager-cainjector -n cert-manager -w
+          kubectl rollout status deployment cert-manager-webhook -n cert-manager -w
+          kubectl rollout status deployment cert-manager -n cert-manager -w
+    
+      - name: Install lifecycle-toolkit
+        run: |
+          kubectl apply -f ~/download/artifacts/keptn-lifecycle-operator-manifest
+          kubectl apply -f ~/download/artifacts/scheduler-manifest
+          kubectl rollout status deployment keptn-scheduler -n keptn-lifecycle-toolkit-system -w
+          kubectl rollout status deployment klc-controller-manager -n keptn-lifecycle-toolkit-system -w
```

Comment:
can we group these steps somehow in a single action since they are shared? 
It would simplify changes and version bump of the cert manager